Jenkinsfile

```diff
@@ -52,6 +52,7 @@ kibanaPipeline(timeoutMinutes: 155, checkPrChanges: true) {
           ]),
         ])
       }
+      githubCommitStatus.onFinish()
```

Comment:
We should add this to all the `Jenkinsfile`s which use `workers.groovy`, right?

Reply:
Only where we want commit statuses to be recorded. It wouldn't be useful several of the jobs (e.g. flaky test suite runner, or the code coverage job)

Reply:
But they're going to get a pending status right?

Reply:
Oh whoops, I see what you're saying. I meant to add a flag to enable `onStart()`, but totally forgot. Going to switch this back to Draft until I add it.